gravitee-apim-repository/gravitee-apim-repository-elasticsearch/src/main/java/io/gravitee/repository/elasticsearch/v4/healthcheck/adapter/AvailabilityQueryMapper.java

```diff
@@ -61,12 +61,14 @@ public Maybe<AvailabilityResponse> adaptResponse(SearchResponse response) {
         if (aggregations == null || aggregations.isEmpty()) {
             return Maybe.empty();
         }
-        long total = response.getSearchHits().getTotal().getValue();
+
         final var entrypointsAggregation = aggregations.get(BY_FIELD_AGGS);
         if (entrypointsAggregation == null) {
             return Maybe.empty();
         }
 
+        var total = entrypointsAggregation.getBuckets().stream().map(jsonNode -> jsonNode.get("doc_count").asLong()).reduce(0L, Long::sum);
```

Comment:
Can you put a comment over this line with a link to the doc?

Reply:
👍